pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -41,7 +41,15 @@ rec {
     settingsSha256 = "0n4pj8dzkr7ccwrn5p46mn59cnijdhg8zmn3idjzrk56pq0hbgjr";
     persistencedSha256 = "0bnjr0smhlwlqpyg9m6lca3b7brl2mw8aypc6p7525dn9d9kv6kb";
 
-    patches = lib.optional (kernel.meta.branch == "4.19") ./drm_mode_connector.patch;
+    patches = [
+    (fetchurl {
+      url = "https://git.archlinux.org/svntogit/packages.git/plain/trunk/kernel-4.16.patch?h=2ad07241ea525a6b6b555b6cb96a97634a4b2cb0";
+      sha256 = "11b3dp0na496rn13v5q4k66bf61174800g36rcwj42r0xj9cfak2";
+    })
+    (fetchurl {
+      url = "https://git.archlinux.org/svntogit/packages.git/plain/trunk/kernel-5.1.patch?h=42d50ef8d6048608d18bdf2c296dd335260c5a1a";
+      sha256 = "03v46ym2bcckg9q2xrilkg21hfiwypr6gl4jmly2q3m4yza9ja6r";
+    })];
```

Comment:
The file should be deleted, right?

Reply:
`pkgs/os-specific/linux/nvidia-x11/drm_mode_connector.patch` didn't apply cleanly anymore. I deleted it now and updated the commit.

I wanted to stick with what archlinux provides, as their patches use `#if LINUX_VERSION_CODE < KERNEL_VERSION(…)` and seem to apply on recent kernels.